interface/billing/apply_payments.php

```diff
@@ -233,7 +232,7 @@ function upload_file_to_client_pdf($file_to_send) {
 
   // Print or download statements if requested.
   //
-if (($_POST['form_print'] || $_POST['form_download'] || $_POST['form_pdf']) || $_POST['form_portalnotify'] && $_POST['form_cb']) {
+  if (($_POST['form_print'] || $_POST['form_download'] || $_POST['form_pdf']) || $_POST['form_portalnotify'] && $_POST['form_cb']) {
```

Comment:
move the ) from the end of $_POST['form_pdf'} to the end of $_POST['form_portalnotify'] or is this in another PR